src/Android/Effects/HintColorSwitchEffect.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.ComponentModel;
+using Android.Graphics.Drawables;
+using Bit.Droid.Effects;
+using Bit.Droid.Utilities;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+[assembly: ExportEffect(typeof(HintColorSwitchEffect), "HintColorSwitchEffect")]
+namespace Bit.Droid.Effects
+{
+    public class HintColorSwitchEffect : PlatformEffect
+    {
+        protected override void OnAttached()
+        {
+            if (Control is Android.Widget.Switch switchView)
+            {
+                switchView.SetHintTextColor(ThemeHelpers.MutedColor);
```

Comment:
If you change themes, does this update correctly dynamically?

Reply:
Removed the effect and added the content to the CustomSwitchRenderer and now is updating dynamically on theme change